src/types/profile.js

```diff
@@ -855,6 +855,16 @@ export type ProfileMeta = {|
   // They also use the "product" field in the meta information, but this is somewhat
   // ambiguous. This field, if present, is unambiguous that it was imported.
   importedFrom?: string,
+
+  // The following are settings that are used to configure the views for
+  // imported profiles, as some features do not make sense for them
+
+  // Do not distinguish between different stack types?
+  usesOnlyOneStackType?: boolean,
```

Comment:
I don't see where this is used. Is this implemented in another PR?

Reply:
In `getProfileUsesMultipleStackTypes` which's return value determines whether the `JavaScript/Native/Combined` radio button is shown.

Reply:
ah right, not sure why I couldn't find it earlier, thanks.

BTW I kind of wonder why we couldn't infer this from the data in a thread actually. For example in a Firefox for Android profile, we will commonly have some Java-only threads and some other mixed native/JS threads, and therefore it would make sense that we can choose this on a single thread rather than on the whole profile. This could be some improvement for a later patch.

Reply:
Inferring needs heuristics and per-thread is more complex (what do you do for merged threads?).